### DISCUSSION TITLE
Non-standard image resolution

### DISCUSSION THREAD

**Discussion by Sachawech** [ImageMagick/ImageMagick#3734] [Development]
Hello!

I love this and use it particularly to export mp4s out of Houdini. It works great and I'm grateful for all the work that's been put into ImageMagick.

Sometimes I want to export sequences that are not 16x9 and then I get an error. As soon as I set the resolution back to 16x9 everything is all good. I was wondering if it would be possible to export something outside that particular ratio? Often I'd like to export things that are 1x1 or even something random like 1653x1212px.

Apologies if this is the wrong place to ask.

Thanks again!

Cheers
Sacha

**Comment by snibgo** [comment]
What version of ImageMagick, on what platform?

> Sometimes I want to export sequences that are not 16x9 and then I get an error.

I don't understand. ImageMagick does not have an "export" facility.

Please show what ImageMagick commands you use.

**Reply by Sachawech** [reply]
Thank you for getting back to me!

I am using ImageMagick 7.0.10 Q16-HDRI (64-bit) on Windows.

I followed this tutorial (it's only 3 minutes):
https://www.youtube.com/watch?v=S3DYOeCs9Jk

Thanks again!

**Comment by snibgo** [comment]
That video does not show any ImageMagick commands.

**Reply by Sachawech** [reply]
I think it's using this internal Houdini node which maybe lists some commands?
https://www.sidefx.com/docs/houdini/nodes/top/imagemagick.html

This might not be an ImageMagick issue after all, but rather maybe it has to do with how Houdini has implemented it?

I don't really know how to use ImageMagick other than to export mp4's through the flipbook in Houdini. I just would love to not be restricted to 16x9 resolutions.

Sorry if this question is entirely stupid haha

Cheers
Sacha

**Comment by fmw42** [comment]
In what way does your video not work? How do you know? What tool are you using to view it? Are you sure you are creating it properly with your Houdini tool? What is the source of the video? Do you have lots of images or are you recording something?

**Reply by Sachawech** [reply]
In Houdini, a 3D software, I am rendering out a sequence of images. I can export this image sequence as mp4 using ImageMagick but only if I use a 16x9 resolution. If I don't use a 16x9 resolution it errors and fails to encode.

**Reply by fmw42** [reply]
Check the limitations of Houdini and its use of ImageMagick. ImageMagick can make any size. But I do not know Houdini or its requirements/limitations.

**Reply by Sachawech** [reply]
Houdini can export any resolution if I pick .mov for instance.

I will look into how ImageMagick is implemented in Houdini. Thanks for the tip!

**Comment by snibgo** [comment]
> I think it's using this internal Houdini node which maybe lists some commands?
https://www.sidefx.com/docs/houdini/nodes/top/imagemagick.html

That web page does not show any ImageMagick commands.

> This might not be an ImageMagick issue after all, but rather maybe it has to do with how Houdini has implemented it?

Probably, yes.
